src/ts/Offers/Offer.tsx

```diff
@@ -3,37 +3,34 @@ import { Link } from 'react-router';
 import { FontIcon } from 'material-ui';
 
 export default function Offer(props) {
-
   const {
-    key,
-    img,
+    url,
     title,
-    price,
     approx,
   } = props.data;
+  const img = props.data.images.main;
```

Comment:
how can we be sure that those props exist?
perhaps `_.get(props, 'data.images.main')` would be safer?